server/save.py

```diff
@@ -294,7 +294,9 @@ def upgrade(version):
             db.execute_sql(
                 'INSERT INTO layer (id, floor_id, name, type_, player_visible, player_editable, selectable, "index") SELECT _layer.id, floor.id, _layer.name, type_, player_visible, player_editable, selectable, _layer."index" FROM _layer INNER JOIN floor ON floor.location_id = _layer.location_id'
             )
-
+        db.execute_sql(
+            'CREATE TABLE IF NOT EXIST "marker" ("id" INTEGER NOT NULL PRIMARY KEY, `uuid` TEXT NOT NULL , `user_id` INTEGER NOT NULL, `room_id` INTEGER NOT NULL, FOREIGN KEY(`uuid`) REFERENCES `shape`(`uuid`) ON DELETE CASCADE, FOREIGN KEY(`room_id`) REFERENCES `room`(`id`) ON DELETE CASCADE, FOREIGN KEY(`user_id`) REFERENCES `user`(`id`) ON DELETE CASCADE)'
+        )
```

Comment:
This needs to move to a separate elif clause

Reply:
As with changelog and menu.vue, I missed a few updates. Will redo this on the latest dev (probably my lack of experience with github as well ;-)

Reply:
This also seems to be still wrong

Reply:
Yes, as said for some reason I didn't see those new updates. Now updated.